data/json/terrain.json

```diff
@@ -1921,6 +1932,7 @@
         "type" : "terrain",
         "id" : "t_fencegate_o",
         "name": "open wooden gate",
+        "description": "A commercial quality gate made of wood with latch system.  The gate is swung open, preventing nothing from passing through.",
```

Comment:
> The gate is swung open, preventing nothing from passing through

"Swung open" is a past tense verb, probably not what you want here. "Wide open," perhaps?

The "preventing nothing from passing" double negative is likely used here for comedic effect but it reads really oddly to me